src/main/java/stream/flarebot/flarebot/commands/music/RandomCommand.java

```diff
@@ -48,7 +48,7 @@ public String getDescription() {
 
     @Override
     public String getUsage() {
-        return "{%}random [amount]";
+        return "`{%}random [amount]`";
```

Comment:
I know it's disabled but we should still provide more usage info here

Reply:
Sure